src/gui/addnewtorrentdialog.cpp

```diff
@@ -181,9 +184,9 @@ void AddNewTorrentDialog::show(QString source, QWidget *parent)
     if (Utils::Misc::isUrl(source)) {
         // Launch downloader
         Net::DownloadHandler *handler = Net::DownloadManager::instance()->downloadUrl(source, true, 10485760 /* 10MB */, true);
-        connect(handler, SIGNAL(downloadFinished(QString, QString)), dlg, SLOT(handleDownloadFinished(QString, QString)));
-        connect(handler, SIGNAL(downloadFailed(QString, QString)), dlg, SLOT(handleDownloadFailed(QString, QString)));
-        connect(handler, SIGNAL(redirectedToMagnet(QString, QString)), dlg, SLOT(handleRedirectedToMagnet(QString, QString)));
+        connect(handler, SIGNAL(downloadFinished(QString,QString)), dlg, SLOT(handleDownloadFinished(QString,QString)));
+        connect(handler, SIGNAL(downloadFailed(QString,QString)), dlg, SLOT(handleDownloadFailed(QString,QString)));
+        connect(handler, SIGNAL(redirectedToMagnet(QString,QString)), dlg, SLOT(handleRedirectedToMagnet(QString,QString)));
```

Comment:
Sorry to pollute your PR, i'll delete this if it turns out to be intentional.
@evsh i find it strange that it removes the space after the comma. Is it intentional? It also removes space before & and * while in the declaration and definition it adds a space.